src/lua-function-list.json

```diff
@@ -0,0 +1,325 @@
+{
```

Comment:
Is this file generated by hand or a script? If by script, we may want to add it to some repo, I think.

Reply:
Generated via a Lua script in Mudlet. I'll put the source up somewhere.

Reply:
:thought_balloon: I guess there may be future scope to scan the functions the user / packages / modules insert into a particular profile - though then you'd need to consider each profile separately rather than a single global autocompleter system in the `mudlet` class - unless you use that separate provider infrastructure to merge the common Mudlet provided functions as one *provider* with each profile having a separate one that is used with the common one when it is the one being considered in the editor...